src/components/LiquidityMining/Create/Steps/PreviewAndCreate/index.tsx

```diff
@@ -43,45 +99,133 @@ export default function PreviewAndCreate({
+      : 10000000
+
+    const baseinUsd = parseFloat(nativeTokenPrice.multiply(nativeCurrencyUSDPrice).toFixed(22))
+
+    let baseValue
+    if (showUSDValue) {
+      baseValue = base
+    } else {
+      baseValue = base / baseinUsd
     }
-    if (userBalance && reward && reward.greaterThan('0') && userBalance.lessThan(reward)) {
-      return 'Insuffucient balance'
+    const tokenOrPair = liquidityPair instanceof Token ? liquidityPair : liquidityPair?.liquidityToken
+
+    if (tokenOrPair && base !== 0 && baseinUsd !== 0) {
+      setSimulatedStakedAmount(
+        parseUnits(
+          calculatePercentage(base / baseinUsd, simulatedValuePercentage).toString(),
+          tokenOrPair.decimals
+        ).toString()
+      )
     }
-    return 'Deposit & create'
-  }
 
+    return calculatePercentage(baseValue, simulatedValuePercentage)
+  }, [
+    setSimulatedStakedAmount,
+    liquidityPair,
+    simulatedValuePercentage,
+    stakingCap,
+    nativeTokenPrice,
+    nativeCurrencyUSDPrice,
+    showUSDValue,
+  ])
+
+  const liquidityPercentChangeCallback = useCallback((value: number) => {
+    setSimulatedValuePercentage(value)
+  }, [])
+
+  const [innerLiquidityPercentage, setInnerLiquidityPercentage] = useDebouncedChangeHandler(
+    simulatedValuePercentage,
+    liquidityPercentChangeCallback,
+    10
+  )
   useEffect(() => {
     setAreButtonsDisabled(
-      !!(!account || !reward || !reward.token || reward.equalTo('0') || (userBalance && userBalance.lessThan(reward)))
+      !!(!account || !reward || !liquidityPair || !startTime || !endTime || approvals || campaign === null)
     )
-  }, [account, reward, userBalance])
+  }, [account, reward, liquidityPair, startTime, endTime, approvals, campaign])
+  const getConfirmButtonMessage = () => {
+    if (!account) return 'Connect your wallet'
+    else if (approvals) return 'Rewards not apporoved'
+    else if (campaign === null) return 'One of tokens not priced'
+
+    return 'Deposit & create'
+  }
+  const isSingleSided = campaign instanceof SingleSidedLiquidityMiningCampaign
 
+  const handleUSDValueClick = useCallback(() => {
+    setShowUSDValue(!showUSDValue)
+  }, [showUSDValue])
   return (
     <Flex flexDirection="column" style={{ zIndex: -1 }}>
+      {campaign !== null && !loadingNativeCurrencyUsdPrice && (
+        <CampaignDetailWrapper flexDirection={'row'} mb="32px">
+          <StyledCampaignCard
+            token0={
+              campaign instanceof SingleSidedLiquidityMiningCampaign
+                ? campaign.stakeToken
+                : campaign.targetedPair.token0
+            }
+            usdLiquidity={getStakedAmountUSD(campaign.staked.nativeCurrencyAmount, nativeCurrencyUSDPrice)}
+            token1={campaign instanceof LiquidityMiningCampaign ? campaign.targetedPair.token1 : undefined}
+            // containsKpiToken={campaign instanceof LiquidityMiningCampaign ? campaign.containsKpiToken : false}
```

Comment:
Undesird comment
```suggestion
```

Reply:
lets leave it for now just as a reminder we are not checking if it has kpi tokens...